extension/src/popup/components/sendPayment/SendConfirm/TransactionDetails/index.tsx

```diff
@@ -297,7 +297,7 @@ export const TransactionDetails = ({ goBack }: { goBack: () => void }) => {
             sourceAsset: sourceAsset.code,
           });
 
-          if (isSoroswap) {
+          if (isSoroswap && destAsset.issuer) {
```

Comment:
only add the token ID if there is a token ID found (in the case of native, this will be skipped)